Fix links (#1615)
openvinotoolkit/model_server: README.md

[Intel Movidius Myriad VPUs](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_MYRIAD.html), 
[GPU](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_GPU.html), and 
[HDDL](https://docs.openvino.ai/2022.2/openvino_docs_OV_UG_supported_plugins_HDDL.html) 

> - works with Bare Metal Hosts as well as [Docker containers](https://docs.openvino.ai/2022.3/ovms_docs_deploying_server.html) 

Reviewer: ```suggestion
- works with [Bare Metal Hosts and Docker containers](https://docs.openvino.ai/2022.3/ovms_docs_deploying_server.html) 
```

* [Model repository configuration](https://docs.openvino.ai/2022.2/ovms_docs_models_repository.html)

> * [Using a docker container](https://docs.openvino.ai/2022.2/ovms_docs_deploying_server.html)

Reviewer: merge those 2 link to one items - Deployment options

Author: Ok.
